src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/httpRenderer.jsx

```diff
@@ -10,7 +10,7 @@ class HttpRenderer extends React.Component {
   };
 
   renderUrl = url => {
-    return url.match(/^https?:\/\//) ? <a href={url}>{url}</a> : <em>{url}</em>;
+    return (typeof url === "string" && url.match(/^https?:\/\//)) ? <a href={url}>{url}</a> : <em>{url}</em>;
```

Comment:
do we need to worry about `url` being undefined or "falsey" at all?

Reply:
The python version does not validate breadcrumb data at all. SDKs can send us any crap. I don't understand how this bug doesn't occur more often.

Reply:
We should fix this in the new Rust normalizer at some point but right now we are focused on getting the behavior as close as possible to Python